applications/ecallmgr/src/ecallmgr_fs_notify.erl

```diff
@@ -124,23 +129,27 @@ notify(Username, Realm, Event) ->
             case ensure_contact_user(Contact, Username, Realm) of
                 'undefined' ->
                     lager:error("invalid contact ~p: ~p", [Contact, Registration]);
-                Valid -> send_notify(Node, Username, Realm, Event, Valid)
+                Valid -> send_notify(Node, Username, Realm, JObj, Valid)
             end
     end.
 
--spec send_notify(atom(), ne_binary(), ne_binary(), ne_binary(), ne_binary()) -> 'ok'.
-send_notify(Node, Username, Realm, Event, Contact) ->
+-spec send_notify(atom(), ne_binary(), ne_binary(), kz_json:object(), ne_binary()) -> 'ok'.
+send_notify(Node, Username, Realm, JObj, Contact) ->
     AOR = To = From = kzsip_uri:ruri(#uri{user=Username, domain=Realm}),
     SIPHeaders = <<"X-KAZOO-AOR : ", AOR/binary, "\r\n">>,
+    Event = kz_json:get_ne_binary_value(<<"Event">>, JObj),
     Headers = [{"profile", ?DEFAULT_FS_PROFILE}
               ,{"contact-uri", Contact}
               ,{"extra-headers", SIPHeaders}
               ,{"to-uri", To}
               ,{"from-uri", From}
               ,{"event-string", Event}
               ],
-    Resp = freeswitch:sendevent(Node, 'NOTIFY', Headers),
-    lager:info("send NOTIFY with Event '~s' to '~s@~s' via ~s: ~p", [Event, Username, Realm, Node, Resp]).
+    Body = kz_json:get_binary_value(<<"Body">>, JObj),
+    ContentType = kz_json:get_binary_value(<<"Content-Type">>, JObj),
+    Resp = freeswitch:sendevent(Node, 'NOTIFY', Headers, Body, ContentType),
```

Comment:
as mentioned below, there is no nered to add `freeswitch:sendevent/5`, filter the headers and send